deploy/helms/legion-core/templates/jenkins/jenkins-server.yaml

```diff
@@ -117,8 +119,10 @@ spec:
         - mountPath: /usr/share/jenkins/ref/init.groovy.d/configure-cloud.groovy
           subPath: configure-cloud.groovy
           name: dynamic-scripts
+        - mountPath: /var/jenkins_home
+          name: jenkins-home
         {{- if .Values.jenkins.git_secret_name }}
-        - mountPath: /var/jenkins_home/.ssh
+        - mountPath: /var/jenkins.ssh
```

Comment:
Why do we need if/parameter here?

Reply:
We can disable using specific git secret (for example, for public repos)

Reply:
Resolved